WordPress/Sniffs/Arrays/ArrayIndentationSniff.php

```diff
@@ -0,0 +1,189 @@
+<?php
+/**
+ * WordPress Coding Standard.
+ *
+ * @package WPCS\WordPressCodingStandards
+ * @link    https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards
+ * @license https://opensource.org/licenses/MIT MIT
+ */
+
+/**
+ * Enforces WordPress array indentation for multi-line arrays.
+ *
+ * @link    https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/#indentation
+ *
+ * @package WPCS\WordPressCodingStandards
+ *
+ * @since   0.12.0
+ */
+class WordPress_Sniffs_Arrays_ArrayIndentationSniff extends WordPress_Sniff {
+
+	/**
+	 * The --tab-width CLI value that is being used.
+	 *
+	 * @var int
+	 */
+	private $tab_width;
+
+
+	/**
+	 * Returns an array of tokens this test wants to listen for.
+	 *
+	 * @return array
+	 */
+	public function register() {
+		return array(
+			T_ARRAY,
+			T_OPEN_SHORT_ARRAY,
+		);
+	}
+
+	/**
+	 * Processes this test, when one of its tokens is encountered.
+	 *
+	 * @param int $stackPtr The position of the current token in the stack.
+	 *
+	 * @return void
+	 */
+	public function process_token( $stackPtr ) {
+		if ( ! isset( $this->tab_width ) ) {
+			$cli_values = $this->phpcsFile->phpcs->cli->getCommandLineValues();
+			if ( ! isset( $cli_values['tabWidth'] ) || 0 === $cli_values['tabWidth'] ) {
+				// We have no idea how wide tabs are, so assume 4 spaces for fixing.
+				$this->tab_width = 4;
+			} else {
+				$this->tab_width = $cli_values['tabWidth'];
+			}
+		}
+
+		/*
+		 * Determine the array opener & closer.
+		 */
+		if ( T_ARRAY === $this->tokens[ $stackPtr ]['code'] ) {
+			if ( ! isset( $this->tokens[ $stackPtr ]['parenthesis_opener'] ) ) {
+				return; // Live coding.
+			}
+			$opener = $this->tokens[ $stackPtr ]['parenthesis_opener'];
+
+			if ( ! isset( $this->tokens[ $opener ]['parenthesis_closer'] ) ) {
+				return; // Live coding.
+			}
+			$closer = $this->tokens[ $opener ]['parenthesis_closer'];
+		} else {
+			// Short array syntax.
+			$opener = $stackPtr;
+
+			if ( ! isset( $this->tokens[ $stackPtr ]['bracket_closer'] ) ) {
+				return; // Live coding.
+			}
+			$closer = $this->tokens[ $stackPtr ]['bracket_closer'];
+		}
+
+		if ( $this->tokens[ $opener ]['line'] === $this->tokens[ $closer ]['line'] ) {
+			// Not interested in single line arrays.
+			return;
+		}
+
+		/*
+		 * Determine the indentation of the line containing the array opener.
+		 */
+		$indentation = '';
+		$column 	 = 1;
```

Comment:
It was & is now fixes.

Yet another thing for which I'm wondering why there is no sniff to auto detect & fix this ;-)